### PR TITLE
EDM-1476: add Events validation TCs

### PR DESCRIPTION
**Pull request by amalykhi**
## Summary by CodeRabbit

* **Tests**
  * Added a comprehensive end-to-end CLI events test suite covering resource lifecycle (create/update/delete), event filtering (including invalid selectors/unknown kinds), pagination/continuation, sorting, validation warnings/errors, deletion events, CLI argument edge cases, and cleanup.
  * Removed legacy event-specific test blocks and simplified other lifecycle tests to use a single standardized success matcher.
* **Chores**
  * Standardized example YAML name constants and consolidated event-reason strings; removed obsolete helpers, flags, and unused imports to simplify test scaffolding.

### CODE REVIEW

**Comment by eldar101** [comment]
if it's just a move, ping me when all checks pass and I'll approve.

**Comment by eldar101** [comment]
Just fix the failures please, otherwise LGTM.

File: test/e2e/cli/cli_events_test.go [hunk 1]
```diff
@@ -0,0 +1,468 @@
...
+
+			By("Querying events with fieldSelector kind=Device")
+			out, err := harness.RunGetEvents(fieldSelector, fmt.Sprintf("%s=%s", kind, util.DeviceResource))
+			Expect(err).ToNot(HaveOccurred())
+			Expect(out).To(MatchRegexp(formatResourceEvent(util.DeviceResource, deviceName, util.EventCreated)))
+
+			By("Querying events with fieldSelector kind=Fleet")
+			out, err = harness.RunGetEvents(fieldSelector, fmt.Sprintf("%s=%s", kind, util.FleetResource))
+			Expect(err).ToNot(HaveOccurred())
+			Expect(out).To(MatchRegexp(formatResourceEvent("Fleet", fleetName, util.EventCreated)))
```
Comment (siserafin) [inline test/e2e/cli/cli_events_test.go]: If you use util.\<resource-name\> I think you should use it everywhere

Comment (amalykhi) [reply test/e2e/cli/cli_events_test.go]: I just moved this test from cli_test, but fixed, also added ResourceCreated event var

Comment (siserafin) [reply test/e2e/cli/cli_events_test.go]: I remember I saw that test before

Comment (amalykhi) [reply test/e2e/cli/cli_events_test.go]: @siserafin  I added 83588 test here
